src/content/blog/2023/hello-nf-core-world.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Hello nf-core world'
+subtitle: 'The inital nf-core blog post'
+headerImage: 'https://images.unsplash.com/photo-1590005354167-6da97870c757'
+headerImageAlt: 'Green apples cut in various ways laying on an bright orange surface'
+pubDate: 2023-11-03T08:00:00.000+01:00
+authors:
+    - 'mashehu'
+    - 'ewels'
+label:
+    - 'community'
+---
+
+## Greetings, nf-core Community
+
+Today marks the launch of the nf-core blog, a new avenue for keeping our community informed and connected.
+This blog will serve as a central hub for in-depth updates and discussions.
+
+## Our Vision for the Blog
+
+This platform is created with the intent to:
+
+-   Share substantive updates from our community’s activities. For example, descriptions of features included in a pipeline release.
+-   Provide clarity on decisions made by the core and maintainers' group.
+-   Offer recaps of nf-core events and hackathons.
+-   Announce and detail updates to nf-core tools.
+
+## Content Snapshot
+
+Expect concise, informative posts covering:
+
+-   **Community News**: Insights into our collective advancements and achievements.
+-   **Technical Insights**: Explanations of decisions and updates from our technical leadership.
+-   **Event Highlights**: Overviews of key takeaways from various nf-core gatherings.
+-   **Tool Updates**: Announcements and insights into the latest nf-core tool releases.
+
+All contributions need to align with the nf-core [code of conduct](https://nf-co.re/code_of_conduct). 
+## Contribution Made Easy
+
+Contributing to this blog is quite straight forward. You just need to add a markdown (or MDX) file to the `src/content/blog` folder.
+The file name title of the post separated by dashes. For example, `hello-nf-core-world.mdx`.
+Fill in the frontmatter (the stuff between the three dashes on the top of the markdown files) and start writing your post.
+
+## Follow Along
+
+Similar to the events page, also our blog posts come with an RSS feed. Subscribe to it at [https://nf-co.re/blog/rss.xml](https://nf-co.re/blog/rss.xml) to stay up to date with the latest posts.
```

Comment:
We should create a new slack channel for this to auto-post to as well, and mention it here.